.github/workflows/deploy.yml

```
@@ -3,45 +3,113 @@ on:
+        uses: actions/checkout@v4
+
+      # Rustバックエンドのビルド
+      - name: Setup Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+
+      - name: Cache Rust dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)